src/_data/LiveData.js

```diff
@@ -133,6 +134,6 @@ export const connect = async store => {
         api.events.on(key, data => store.dispatch(action(data)));
         api.events.on(key, () => window.console.warn);
     });
-
+    api.getResidences();
```

Comment:
@borisyankov where do we need to change ? I think if you agree with qingwei then you agree with me by extension. 😉